Earthfile

```diff
@@ -211,23 +211,24 @@ test-all:
     BUILD ./examples/tests+experimental
 
 examples:
-    BUILD ./examples/go+docker
-    BUILD ./examples/java+docker
-    BUILD ./examples/js+docker
     BUILD ./examples/cpp+docker
-    BUILD ./examples/scala+docker
     BUILD ./examples/dotnet+docker
-    BUILD ./examples/python+docker
-    BUILD ./examples/ruby+docker
-    BUILD ./examples/ruby-on-rails+docker
     BUILD ./examples/elixir+docker
+    BUILD ./examples/go+docker
+    BUILD ./examples/grpc+test
```

Comment:
this is the only addition, the rest of the change is just sorting these alphabetically.